src/app/layout.tsx

```diff
@@ -33,7 +33,9 @@ const RootLayout = async ({ children }: { children: React.ReactNode }) => {
                     >
                       <MainHeader />
                     </Box>
-                    <Box sx={{ padding: '40px' }}>{children}</Box>
+                    <Box sx={{ padding: '40px' }}>
+                      <Box sx={{ margin: '10px 90px' }}>{children}</Box>
+                    </Box>
```

Comment:
```suggestion
                    <Box sx={{ padding: '40px', margin: '10px 90px' }}>{children}</Box>
```

Can this be condensed to one like this?